libvast/src/system/node.cpp

```diff
@@ -381,13 +375,22 @@ void node_state::init(std::string init_name, path init_dir) {
   name = std::move(init_name);
   dir = std::move(init_dir);
   // Bring up the tracker.
-  tracker = self->spawn<monitored>(system::tracker, name);
-  self->set_down_handler([=](const down_msg& msg) {
-    VAST_IGNORE_UNUSED(msg);
-    VAST_DEBUG(self, "got DOWN from", msg.source);
-    self->quit(msg.reason);
+  tracker = self->spawn<linked>(system::tracker, name);
+  self->set_exit_handler([=](const exit_msg& msg) {
+    VAST_DEBUG(self, "got EXIT from", msg.source);
+    std::vector<caf::actor> actors;
+    if (auto acc = self->system().registry().get(atom::accountant_v))
+      actors.push_back(caf::actor_cast<caf::actor>(acc));
```

Comment:
We could mention in a comment here that we intentionally use a sequential shutdown policy and put the accountant first to allow it to clear its buffers before the index, archive, etc. shut down.